projects/igniteui-angular/src/lib/grid/column-pinning.component.ts

```diff
@@ -41,16 +34,11 @@ export class IgxColumnPinningComponent extends ColumnChooserBase {
     constructor(public cdr: ChangeDetectorRef) {
         super(cdr);
     }
-    @Output()
-    public onColumnPinning = new EventEmitter<IPinColumnEventArgs>();
```

Comment:
Should this event emitter really go away?

Reply:
It wasn't used at all because we already emit such an event when setting the pinned property.